### DISCUSSION TITLE
Placement Blockages

### DISCUSSION THREAD

**Discussion by GundraHarshini** [The-OpenROAD-Project/OpenROAD#6429] [General]
Is there any direct command to place blockages in specific coordinates, If no whats the way do that. I have referred to plcement_blockages.tcl in scripts, In that odb::dbBlockage_create is used

**Comment by maliberty** [comment]
There is no higher level command for it currently.

**Reply by rovinski** [reply]
Would make a good issue for a newcomer.

**Reply by rovinski** [reply]
#6449
